docs/_toc.yml

```diff
@@ -46,6 +46,7 @@ parts:
     - file: photonics/examples/refinement.py
     - file: electronics/examples/capacitor.py
     - file: electronics/examples/coax_cable.py
+    - file: electronics\examples\RF CPW transmission line tutorial\RF CPW transmission line tutorial.py
```

Comment:
one more thing I noticed after checking out the branch: I would avoid whitespace in directory and filenames. Could we use underscores?

Reply:
Done :)